context-menu-demo/_locales/en/messages.json

```diff
@@ -0,0 +1,42 @@
+{
+  "extensionName": {
+    "message": "Context menu demo",
+    "description": "Name of the extension."
+  },
+
+  "extensionDescription": {
+    "message": "Demonstrates the contextMenus API.",
+    "description": "Description of the add-on."
+  },
+
+  "contextMenuItemSelectionLogger": {
+    "message": "Log '%s' to the console",
```

Comment:
This logs to the browser console, in my testing, perhaps we should we specify which console it goes to.